homeassistant/components/epion/config_flow.py

```diff
@@ -0,0 +1,62 @@
+"""Config flow for Epion."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from epion import Epion, EpionAuthenticationError, EpionConnectionError
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import CONF_API_KEY
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class EpionConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Epion."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle a flow initiated by the user."""
+        errors = {}
+        if user_input:
+            try:
+                key_valid = await self.hass.async_add_executor_job(
+                    self._check_api_key, user_input[CONF_API_KEY]
+                )
```

Comment:
Can we also get some kind of unique identifier for the user? This way we can make sure the user can't setup the same integration twice